### PR TITLE
feat: pointOnColumn options

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* apply `pointOnColumn` 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jung-han** [comment]
ok to test

File: src/component/boxSeries.ts [hunk 1]
```diff
@@ -97,14 +95,11 @@ export default class BoxSeries extends Component {
...
-    const valueLabels = axes[this.valueAxis].labels;
-    const tickDistance = this.getTickDistance(axes[this.labelAxis].validTickCount);
-
     const seriesModels: RectModel[] = this.renderSeriesModel(
       seriesData,
       colors,
-      valueLabels,
-      tickDistance
+      axes[this.valueAxis].labels,
+      axes[this.labelAxis].tickDistance
```
Comment (jungeun-cho) [inline src/component/boxSeries.ts:102]: 이부분도 axes모듈에서 계산된 값을 사용하면 될 것 같습니다.
제가 수정해서 커밋해놓았어요. :)

File: src/component/boxStackSeries.ts [hunk 2]
```diff
@@ -40,15 +40,13 @@ export default class BoxStackSeries extends BoxSeries {
...
-    const tickDistance = this.getTickDistance(axes[this.labelAxis].validTickCount);
 
     const seriesModels: RectModel[] = this.renderStackSeriesModel(
       stackType,
       seriesData,
       colors,
-      valueLabels,
-      tickDistance
+      axes[this.valueAxis].labels,
+      axes[this.labelAxis].tickDistance
```
Comment (jungeun-cho) [inline src/component/boxStackSeries.ts:49]: stack도 동일하게 axes에서 계산된 tickDistance값을 사용하도록 합니다.

File: tests/components/boxSeries.spec.ts [hunk 3]
```diff
@@ -26,12 +26,11 @@ const chartState = {
...
     xAxis: {
       labels: [0, 5],
-      tickCount: 2,
-      validTickCount: 2
+      tickCount: 2
     },
     yAxis: {
       pointOnColumn: true,
-      validTickCount: 2
+      tickDistance: 40
```
Comment (jungeun-cho) [inline tests/components/boxSeries.spec.ts:33]: 테스트 코드도 함께 수정해놓았습니다.

**Pull request merged**